tests/unit_tests/gui/test_gui_load.py

```diff
@@ -189,7 +133,9 @@ def test_that_the_ui_show_no_warnings_when_observations_found(qapp):
         assert gui.windowTitle() == "ERT - poly.ert"
 
 
-def test_that_the_ui_show_warnings_when_there_are_no_observations(qapp, tmp_path):
+def test_gui_shows_a_warning_and_disables_update_when_there_are_no_observations(
```

Comment:
Im confused regarding this `update`. This test verifies that the simulation mode is selectable (or not) based on observations found. Is update referring to es-mda, es and ies?

Reply:
Yes, I guess update is somewhat misleading, it was indeed referring to es-mda, es and ies which performs an "update" or "analysis".